# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/lvp/lvp/hotfix.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

This hotfixes a bug in xfree? freeglut? one of those.
If no Window Manager is loaded, the initial Reshape is not called
resulting in no output on the screen.

How did this work before? No idea...


diff -pruN lvp_0.3.2/src/lvp.cpp lvp_0.3.2_patched/src/lvp.cpp
--- lvp_0.3.2/src/lvp.cpp	2004-03-03 16:59:48.000000000 +0100
+++ lvp_0.3.2_patched/src/lvp.cpp	2004-09-28 09:06:12.415815920 +0200
@@ -967,6 +967,7 @@ int main(int argc, char *argv[]){ // {{{
 	window.AddObject(&no);
 
 	glutFullScreen();
+	Reshape(WindW, WindH);
 	glEnable(GL_DEPTH_TEST);
 	glEnable(GL_TEXTURE_2D);
 	glutMainLoop();
